internal/pkg/skuba/upgrade/node/versions_test.go

```diff
@@ -219,11 +219,14 @@ func versionInquirer(versions ...string) kubernetes.VersionInquirer {
 				ContainerRuntimeVersion: version,
 			},
 			ComponentContainerVersion: kubernetes.ComponentContainerVersion{
-				kubernetes.Hyperkube: &kubernetes.ContainerImageTag{Name: "hyperkube", Tag: fmt.Sprintf("v%s", version)},
-				kubernetes.Etcd:      &kubernetes.ContainerImageTag{Name: "etcd", Tag: "3.3.11"},
-				kubernetes.CoreDNS:   &kubernetes.ContainerImageTag{Name: "coredns", Tag: "1.2.6"},
-				kubernetes.Pause:     &kubernetes.ContainerImageTag{Name: "pause", Tag: "3.1"},
-				kubernetes.Tooling:   &kubernetes.ContainerImageTag{Name: "skuba-tooling", Tag: "0.1.0"},
+				kubernetes.APIServer:         &kubernetes.ContainerImageTag{Name: "hyperkube", Tag: fmt.Sprintf("v%s", version)},
+				kubernetes.ControllerManager: &kubernetes.ContainerImageTag{Name: "hyperkube", Tag: fmt.Sprintf("v%s", version)},
+				kubernetes.Scheduler:         &kubernetes.ContainerImageTag{Name: "hyperkube", Tag: fmt.Sprintf("v%s", version)},
+				kubernetes.Proxy:             &kubernetes.ContainerImageTag{Name: "hyperkube", Tag: fmt.Sprintf("v%s", version)},
```

Comment:
Do we have to compare
if version >= 1.18, then
```
kubernetes.APIServer:         &kubernetes.ContainerImageTag{Name: "api-server", Tag: fmt.Sprintf("v%s", version)},
kubernetes.ControllerManager: &kubernetes.ContainerImageTag{Name: "controllermanager", Tag: fmt.Sprintf("v%s", version)},
kubernetes.Scheduler:         &kubernetes.ContainerImageTag{Name: "scheduler", Tag: fmt.Sprintf("v%s", version)},
kubernetes.Proxy:             &kubernetes.ContainerImageTag{Name: "proxy", Tag: fmt.Sprintf("v%s", version)},
```
if version <= 1.17, then
```
kubernetes.APIServer:         &kubernetes.ContainerImageTag{Name: "hyperkube", Tag: fmt.Sprintf("v%s", version)},
kubernetes.ControllerManager: &kubernetes.ContainerImageTag{Name: "hyperkube", Tag: fmt.Sprintf("v%s", version)},
kubernetes.Scheduler:         &kubernetes.ContainerImageTag{Name: "hyperkube", Tag: fmt.Sprintf("v%s", version)},
kubernetes.Proxy:             &kubernetes.ContainerImageTag{Name: "hyperkube", Tag: fmt.Sprintf("v%s", version)},
```

Reply:
Not really, these are just placeholders. As long as the map contains the component needed for the unit tests, I think we are good. The image name doesn't really matter, we don't test it. 

IMO, assuming we did (I think it would be error prone and therefore a fragile test), we would need to make a different test for it, ensuring each component contains the expected container image for specific versions.